kolibri/plugins/learn/assets/src/views/LearnAppBarPage.vue

```diff
@@ -42,6 +42,12 @@
         default: null,
       },
     },
+    $trs: {
+      learnPageMenuLabel: {
+        message: 'Learn page menu',
```

Comment:
And I also don't see this new ``aria-label`` 😕 

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_015](https://user-images.githubusercontent.com/1457929/186912878-b3af1c22-80cb-444a-8a98-e184fac9aa28.png)